### PR TITLE
WDP200303-1

### PR DESCRIPTION
**Pull request by ichtys77**
I add 
- photos do product card,
- props 'image' to initialState
- rendering examples photos in ProductBox component,
- at last i add "img" styling

### CODE REVIEW

**Comment by Varenthein** [comment]
The title of task should be the same as task id. So in your case, we should name it WDP200303-1. Please fix that.

Also, please provide short description of "What is the subject of the task?" and "How I've done it?".

File: src/components/common/ProductBox/ProductBox.module.scss [hunk 1]
```diff
@@ -26,11 +26,25 @@
...
+        z-index: 1;
+      }
+    }
+
+    img {
+      position: absolute;
+      top: 0;
+      left: 1px;
+      right: 1px;
+      max-width: calc(100% - 2px);
```
Comment (Varenthein) [inline src/components/common/ProductBox/ProductBox.module.scss]: If you set one of pair left/right, the second on doesn't really matter. Test it yourself. If you set right at the level of 100 or 200 pexels it won't do anything. The same goes for top/bottom pair.

Another thing is calc width. Maybe you could deal without it? In a simpler manner?

Comment (ichtys77) [reply src/components/common/ProductBox/ProductBox.module.scss]: i fix it... i suppose...

**Comment by Varenthein** [comment]
That's okay now, but two requests from me.

When you fix your task according to requested changes, please do two things:
1. Remove `requested changes` label from the PR
2. Move the task to "In review" column in Jira

You can remove the label by clicking "Settings" icon on labels:

image

...and then clicking on the label you want to remove:

image

**Comment by ichtys77** [comment]
Yesss... my Master...

**Pull request merged**